src/database.js

```diff
@@ -1,10 +1,18 @@
 var mysql = require('mysql');
+var config = require('dotenv').config();
 
-var con = mysql.createConnection({
+var connectionString = {
   host: "localhost",
-  user: "yourusername",
-  password: "yourpassword"
-});
+  user: "root",
+  password: "root"
+};
+
+if (config.parsed) {
+  connectionString.user = config.parsed.DB_USER;
+  connectionString.password = config.parsed.DB_PASS;
+}
+
+var con = mysql.createConnection(connectionString);
```

Comment:
use OR operator as the same can be done in short.

Reply:
thanks for the feedback!